ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -237,89 +297,48 @@ class AdminWorkspaceImpl extends React.Component<UrlParamsProps, State> {
           </div>
           <h3>Cohort Builder</h3>
           <div className='cohort-builder' style={{marginTop: '1rem'}}>
-            {
-              this.workspaceInfoField(
-                '# of Cohorts',
-                resources.workspaceObjects.cohortCount
-              )
-            }
-            {
-              this.workspaceInfoField(
-                '# of Concept Sets',
-                resources.workspaceObjects.conceptSetCount
-              )
-            }
-            {
-              this.workspaceInfoField(
-                '# of Data Sets',
-                resources.workspaceObjects.datasetCount
-              )
-            }
+            <WorkspaceInfoField labelText={'# of Cohorts'}>{resources.workspaceObjects.cohortCount}</WorkspaceInfoField>
+            <WorkspaceInfoField labelText={'# of Concept Sets'}>{resources.workspaceObjects.conceptSetCount}</WorkspaceInfoField>
+            <WorkspaceInfoField labelText={'# of Data Sets'}>{resources.workspaceObjects.datasetCount}</WorkspaceInfoField>
           </div>
           <h3>Cloud Storage Objects</h3>
           <div className='cloud-storage-objects' style={{marginTop: '1rem'}}>
-            {
-              this.workspaceInfoField(
-                'GCS bucket path',
-                resources.cloudStorage.storageBucketPath
-              )
-            }
-            {
-              this.workspaceInfoField(
-                '# of Notebook Files',
-                resources.cloudStorage.notebookFileCount
-              )
-            }
-            {
-              this.workspaceInfoField(
-                '# of Non-Notebook Files',
-                resources.cloudStorage.nonNotebookFileCount
-              )
-            }
-            {
-              this.workspaceInfoField(
-                'Storage used (bytes)',
-                resources.cloudStorage.storageBytesUsed
-              )
-            }
+            <div style={{color: colors.warning, fontWeight: 'bold'}}>
+              NOTE: these may be undercounts, because we process only a single page of storage list results.
```

Comment:
I would clarify in this note that it may undercount if there are ~1000 or more objects in the bucket.